sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/util/DbUtils.java

```diff
@@ -66,6 +67,13 @@
   protected static final Logger log = LoggerFactory.getLogger(DbUtils.class);
   private static final HashCodeCacheKeyGenerator hashCodeCacheKeyGenerator = new HashCodeCacheKeyGenerator();
 
+  public static String convertStringToSearchQuery(String input) {
+    if (input == null) {
+      return "%";
+    }
+    return "%" + input.trim().toUpperCase().replaceAll("[\\s,%]+", " ").replaceAll("_", Matcher.quoteReplacement("\\_")) + "%";
```

Comment:
The % should be escaped to \\% just like the _ character. It is possible to use % as a character in some searchable fields.